debian/rules

```diff
@@ -62,8 +63,12 @@ override_dh_install:
 	:
 	rm -f debian/tmp/etc/grub.d/15_ostree
 	rm -f debian/tmp/usr/lib/libostree/grub2-15_ostree
+	rm -f debian/tmp/usr/lib/tmpfiles.d/ostree-tmpfiles.conf
```

Comment:
I think we may actually want to ship this. All it does is cleanup the left behind overlays at boot from `ostree admin unlock` (non-hotfix mode).

Reply:
Fix pushed.